plugins/epos/plugin.py

```diff
@@ -263,20 +269,40 @@ def rda_a1_02m(self):
         msg
             Message with the results or recommendations to improve this indicator
         """
-        # 2 - Look for the metadata terms in HTML in order to know if they can be accessed manually
 
-        return (0, "")
+        msg = "The link to the manual obtention  of the metadata is " + str(
```

Comment:
Need to check first that `self.metadata_access_manual` is not empty before assigning the check 100 points